### PR TITLE
bug 1585410 - Implement deletion-request ping

### PR DESCRIPTION
**Pull request by chutten**
Checklist for reviewer:

- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [x] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

For glean changes:
- [ ] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

**Comment by jklukas** [comment]
Looks like `deletion-request.4.noclientid.fail.json` is not passing tests.

File: validation/telemetry/deletion-request.4.payload.fail.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "buildId": "20180709105613",
+    "name": "Firefox",
+    "version": "63.0a1",
+    "displayVersion": "63.0a1",
+    "vendor": "Mozilla",
+    "platformVersion": "63.0a1",
+    "xpcomAbi": "x86_64-gcc3",
+    "channel": "default"
+  },
+  "payload": {
```
Comment (jklukas) [inline validation/telemetry/deletion-request.4.payload.fail.json]: This test should include a clientId to make sure it's the payload part that's causing a failure.

File: templates/telemetry/deletion-request/deletion-request.4.schema.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      "maximum": 4
+    }
+  },
+  "required": [
+    "application",
+    "creationDate",
+    "id",
+    "type",
+    "version"
+  ],
```
Comment (jklukas) [inline templates/telemetry/deletion-request/deletion-request.4.schema.json]: Looks like clientId needs to be included here.

**Comment by chutten** [comment]
Sorry for the CI bouncing. Tests aren't working locally on Windows since I don't have Docker (long story elided for brevity's sake), so I've been relying on CI to catch my mistakes.

Should be good for a look now.

**Pull request merged**